annotations/models.py

```diff
@@ -1020,6 +1037,17 @@ class RelationTemplate(models.Model):
     expression = models.TextField(null=True)
     """Pattern for representing the relation in normal language."""
 
+    default_mapping = models.TextField(blank=True, null=True)
```

Comment:
this should be a relationsship to a `DefaultMapping` object that has six properties:
- `subject_type`: relationship node or uri
- `subject_value`: the node reference or a URI
- `predicate_type`: relationship or uri
- ... etc, etc